readline/readline.go

```diff
@@ -786,19 +814,23 @@ func (rl *Instance) escapeSeq(r []rune) {
 		rl.viDeleteByAdjust(rl.viJumpB(tokeniseLine))
 		rl.updateHelpers()
 
-	case seqCtrlDelete, seqCtrlDelete2:
+	case seqCtrlDelete, seqCtrlDelete2, seqAltD:
 		if rl.modeTabCompletion {
 			rl.resetVirtualComp(false)
 		}
-		// This is only available in Insert mode
-		if rl.modeViMode != VimInsert {
-			return
-		}
```

Comment:
any reason for this?

Reply:
Stupidity probably, prob need to readd that to seqaltdelete then as well